### ISSUE TITLE
Change all code examples to UPDATE statements not SELECT

### ISSUE THREAD

**Issue by andrewxhill** [CartoDB/bigmetadata#14]
In the catalog, all the statements should be examples of updates, not selects. That way people can grab the data without hitting their quota ever again.

**Comment by ohasselblad** [comment]
And for table-returning functions, we may want to updates like:
```sql
INSERT INTO testing_boundaries(the_geom, geoid) 
SELECT the_geom, geom_refs 
FROM cdb_observatory.OBS_GetBoundariesByGeometry(
  ST_MakeEnvelope(-74.255735, 40.496044, -73.700272, 40.915256, 4326),
  'us.census.tiger.county')
```

**Comment by andrewxhill** [comment]
pretty nice

**Comment by talos** [comment]
Resolved with #22.
